spec/spec_helper.rb

```diff
@@ -75,3 +91,19 @@ def in_tmp_dir
     end
   end
 end
+
+private
+
+def stringify_for_fastlane(hash)
+  hash.map do |key, value|
+    # rubocop:disable Style/CaseLikeIf
+    if value.is_a?(Hash)
+      "#{key}: {\n#{stringify_for_fastlane(value)}\n}"
+    elsif value.is_a?(String)
+      "#{key}: \"#{value}\""
+    else
+      "#{key}: #{value}"
+    end
+    # rubocop:enable Style/CaseLikeIf
```

Comment:
I tried to use `case-when`, but I couldn't find a neat way to put `value` in the `case` call, because I need to call `is_a?` on it. The only thing I could do was:

```ruby
case
when value.is_a?(Hash)
  ...
when ...
  ...
else
  ...
end
```

Which raised a different lint error saying I should have used `if-elsif` instead 🙃 ♻️ 

Keen to know if there's a different way to write this.

Reply:
```ruby
case value
    when Hash
      puts 'Hash!'
    when String
      puts 'String!'
    else
      puts "What's that?"
    end
```